content/swiftui/concepts/viewmodifier/terms/underline/underline.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '.underline()'
+Description: 'A ViewModifier that applies an underline style to text characters in a View.'
+Subjects:
+   - 'Mobile Development'
+   - 'iOS'
+Tags:
+   - 'SwiftUI'
+   - 'SwiftUI Modifiers'
+CatalogContent:
+   - 'learn-swift'
+   - 'paths/build-ios-apps-with-swiftui'
+---
+
+**.underline()** is a [`ViewModifier`](https://www.codecademy.com/resources/docs/swiftui/viewmodifier) that applies an underline style to text characters in a [`View`](https://www.codecademy.com/resources/docs/swiftui/views).
+## Syntax
+
+```psuedo
+Text("This text will be underlined")
+    .underline()
+```
+
+## Example
```

Comment:
We like descriptive text around the example block. Again, you can look at the published ViewModifier entries to get an idea what we'd like.

Reply:
Example block descriptive text added as per existing ViewModifier entries examples.